e2e/assets/luet-amd64.yaml

```diff
@@ -0,0 +1,15 @@
+general:
+  debug: false
+  spinner_charset: 9
+logging:
+  enable_emoji: false
+repositories:
+  - name: "kairos"
+    description: "kairos repository"
+    type: "docker"
+    cached: true
+    enable: true
+    priority: 2
+    urls:
+      - "quay.io/kairos/packages"
+    reference: 20240215113356-repository.yaml
```

Comment:
is there a way we can have these files an the ones in root in sync? sounds like it will be messy

Reply:
removing the end2end tests? :P

Reply:
created them as soft links so we only have to maintain 1 set of repos

Reply:
well crap, thats not possible of course :(

Reply:
yeah, the way I solved this on kairos repo was to put them in the deeper directory, but it's a crapy solution